FordLabs/PeopleMover
# Add created and last modified fields to user space mapping

## Issue
Resolves #554 

## What was done
- [x] Adds user and timestamp tracking to the user_space_mapping so we know when and who created or updated an entry

## How to test
1. Create a space or invite a user to a space and check the database. The `user_space_mapping` table should have `created_date`, `created_by`, `last_modified_date`, and `last_modified_by` added and populated for the corresponding entry.

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch
